experimental/embeddings/scripts/evaluate_sequence_tagging.py

```diff
@@ -0,0 +1,91 @@
+import argparse
+import os
+import pickle
+
+import flair
+import torch
+from flair.embeddings import TransformerWordEmbeddings
+
+from experimental.embeddings import datasets
+from experimental.embeddings.tasks import sequence_tagging as st
+
+
+def get_args() -> argparse.Namespace:
```

Comment:
so we do not want to use fire or other simple tool?

Reply:
It is temporal solution. I will refactor this as soon as we decide which library we will use for that purpose.